src/misc/support.cpp

```diff
@@ -202,9 +202,22 @@ void FILE_closer::operator()(FILE *f) noexcept
 	}
 }
 
+FILE* open_file(const char* filename, const char* mode)
+{
+#if defined(_WIN32) && defined(__STDC_WANT_SECURE_LIB__)
+	FILE* f;
+	if (0 != fopen_s(&f, filename, mode)) {
+		return nullptr;
+	}
+#else
+	return fopen(filename, mode);
+#endif
+	return nullptr; // just to make GCC happy
+}
```

Comment:
This post-pre-processor chunk is the default return value when Windows `if` check fails (and needs to return something from that path), so suggest snipping this comment.